functional-tests/www/index.js

```diff
@@ -2,51 +2,57 @@ const express = require('express');
 const nunjucks = require('nunjucks');
 const path = require('path');
 const app = express();
-const { xlsxDataRouting } = require('./jsonData/xlsxData.router');
+const {xlsxDataRouting} = require('./jsonData/xlsxData.router');
```

Comment:
I think that there is inconsistency in formatting. Please check if prettier check files in functional-tests

Reply:
ok

Reply:
@tgorskievo only `src/**/*.ts` here